promql/testdata/native_histograms.test

```diff
@@ -0,0 +1,207 @@
+# Minimal valid case: an empty histogram.
+load 5m
+	empty_histogram{}	{{}}
+
+eval instant at 5m histogram_count(empty_histogram{})
+	{} 0
+
+eval instant at 5m histogram_sum(empty_histogram{})
+	{} 0
+
+eval instant at 5m histogram_fraction(-Inf, +Inf, empty_histogram{})
+	{} NaN
+
+eval instant at 5m histogram_fraction(0, 8, empty_histogram{})
+	{} NaN
```

Comment:
Maybe do an `eval instant at 5m empty_histogram` to prove that the histogram is even there.

Reply:
Looks like the parser doesn't support this yet. I added some todo's with the failing tests in 3eb0735a7 against all suggestions of full histogram queries.

Reply:
Added in 65cbd1932